packages/frontend/src/components/AudioRecorder/styles.module.scss

```
@@ -0,0 +1,132 @@
+.microphoneButton {
+  height: 40px;
+  width: 40px;
+  padding-left: 2px;
+  padding-bottom: 8px;
+  background-color: var(--composerBg);
+  border: none;
+
+  &:focus {
+    outline: none;
+  }
+  span {
+    display: block;
+    width: 25px;
+    height: 25px;
+    margin-top: calc((40px - 25px) / 2);
+    -webkit-mask-image: url(./images/microphone.svg);
+    -webkit-mask-size: contain;
+    background-color: var(--composerPlaceholderText);
+    &:hover {
+      background-color: var(--buttonSecondaryText);
+    }
+  }
+}
+
+.audioRecorder {
+  display: flex;
+  flex: 1;
+  flex-direction: row;
+  align-items: stretch;
+  flex-grow: 1;
+  margin-left: 10px;
+  p {
+    margin: 12px 0 0 3px;
+  }
+  .microphoneButton {
+    span {
+      animation: pulse 1.5s infinite ease-in-out;
+      @keyframes pulse {
+        0% {
+          background-color: #eb4c4c58;
+        }
+        50% {
+          background-color: #d70b0b;
+        }
+        100% {
+          background-color: #eb4c4c58;
+        }
+      }
+    }
+  }
+  button:not(.microphoneButton) {
+    padding: 4px;
+    height: 32px;
+    margin-top: 4px;
+    width: 85px;
+    border-radius: 5px;
+    border: none;
+    margin-left: 16px;
+
+    &.stopRecording {
+      background-color: var(--buttonPrimaryBackground);
+      color: #fff;
+      &:hover {
+        background-color: var(--buttonPrimaryHover);
+      }
+    }
+
+    &.cancel {
+      background-color: var(--buttonSecondaryBackground);
+      color: var(--buttonSecondaryText);
+      border: 1px solid #ccc;
+      &:hover {
+        background-color: var(--buttonSecondaryHover);
+      }
+    }
+  }
+}
+
+.timer {
+  margin-top: 12px;
+}
+
+.volumeBar {
+  display: flex;
+  flex: 1;
+  position: relative;
+  display: flex;
+  margin: 15px 20px 0 15px;
+  height: 12px;
+  max-width: 250px;
+  min-width: 60px;
+
+  .mask {
+    display: flex;
+    width: 100%;
+    z-index: 10;
+    height: 12px;
+    position: absolute;
+    left: 0;
+
+    .step {
+      display: flex;
+      width: 10%;
+      height: 10px;
+      border: 1px solid #666;
+    }
```

the line between the segments have the double thickness, which does not look good.

also, the lines better pick up a shading that exist and is more decent, to fit into the rest of the bar.

```suggestion
      border-left: 1px solid #bbb;
      border-top: 1px solid #bbb;
      border-bottom: 1px solid #bbb;
    }

    .step:last-child {
      border-right: 1px solid #bbb;
    }
```

^^ maybe that can be done in a smarter way, but this does the job, is easy to understand, and keeps things together

---

Done